.travis.yml

```diff
@@ -8,6 +8,8 @@ matrix:
     - python: 3.5
       env: NUMPY_VERSION="<1.14.0"
     - python: 3.6
+    - python: 3.7
+      dist: xenial
```

Comment:
I wasn't aware that you needed this, thanks!

Reply:
Yes, see https://github.com/travis-ci/travis-ci/issues/9069 / https://github.com/travis-ci/travis-ci/issues/9815 for details (`sudo` is no longer needed).